SDPi_Supplement/asciidoc/volume0/tf0-main.adoc

```diff
@@ -1,4 +1,37 @@
-= IHE Technical Frameworks General Introduction
+= Volume 0 -- IHE Technical Frameworks General Introduction
 
 NOTE:  This document integrates all the content that makes up the supplement volume 0 (IHE General Introduction) document.
 
+OTE:  This document integrates all the content that makes up the supplement volume 0 document.
+
+IMPORTANT:  Migrate the content below from main.adoc into this document OR rename main.adoc to this ... though the content is different.  +
+{empty} +
+*_See the Word document for what this is supposed to look like_*.
+
+
+[#clause-general]
```

Comment:
A dangling block that should be merged with the subsequent one - and should be named according to our naming conventions:

```asciidoc
[#vol0_clause_general,sdpi_offset=clear]
== IHE Technical Frameworks General Introduction
```

Reply:
Agreed ... see previous query as to where to do the update.

Reply:
Changes made as requested.